README.md

```diff
@@ -79,6 +94,15 @@ resources               clusterpedia.io/v1beta1  false   Resources
 More information about [Search Conditions](https://clusterpedia.io/docs/usage/search/),
 about [Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector) and [Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector)
 
+## Usage Samples
+We can search for resources configured in *PediaCluster*, Clusterpedia supports two types of resource search:
```

Comment:
You can search for resources configured in *PediaCluster*. Clusterpedia supports two resource types: